lerna.json

```diff
@@ -1,5 +1,4 @@
 {
-  "lerna": "2.11.0",
```

Comment:
I don't know why I made this change.  I'll revert this but we should maybe update our version of lerna which no longer requires this.